### ISSUE TITLE
The command defined in "Knp\Command\Twig\DebugCommand" cannot have an empty name. in /var/www/yourist/vendor/symfony/console/Application.php:450

### ISSUE THREAD

**Issue by OtezVikentiy** [KnpLabs/ConsoleServiceProvider#45]
```
PHP Fatal error:  Uncaught Symfony\Component\Console\Exception\LogicException: The command defined in "Knp\Command\Twig\DebugCommand" cannot have an empty name. in /var/www/yourist/vendor/symfony/console/Application.php:450
Stack trace:
#0 /var/www/yourist/vendor/knplabs/console-service-provider/Knp/Provider/ConsoleServiceProvider.php(49): Symfony\Component\Console\Application->add(Object(Knp\Command\Twig\DebugCommand))
#1 /var/www/yourist/vendor/pimple/pimple/src/Pimple/Container.php(118): Knp\Provider\ConsoleServiceProvider->Knp\Provider\{closure}(Object(Silex\Application))
#2 /var/www/yourist/app/bootstrap/bootstrap-cli.php(36): Pimple\Container->offsetGet('console')
#3 /var/www/yourist/bin/run(11): require('/var/www/youris...')
#4 {main}
  thrown in /var/www/yourist/vendor/symfony/console/Application.php on line 450
```

I have such an issue, when i run composer update

**Comment by GregOriol** [comment]
@OtezVikentiy The issue has already been fixed and pull here: https://github.com/KnpLabs/ConsoleServiceProvider/pull/40 but not yet released, see https://github.com/KnpLabs/ConsoleServiceProvider/issues/44

**Comment by OtezVikentiy** [comment]
@GregOriol thanks for reply ))) And when is it going to be merged? Do you know? )))

**Comment by GregOriol** [comment]
It has been merged into master (see the PR 40), but not released yet (see the issue 44) so either you use master or you add a +1 to #44 so that the maintainer releases a new version ;-)

**Comment by OtezVikentiy** [comment]
@GregOriol Done +1 )))

**Comment by skalpa** [comment]
Fixed in v2.2.0 of the service provider.
